airflow/models/dagbundle.py

```diff
@@ -41,3 +47,8 @@ class DagBundleModel(Base):
 
     def __init__(self, *, name: str):
         self.name = name
+
+    @staticmethod
+    @provide_session
+    def get(name: str, session: Session = NEW_SESSION) -> DagBundleModel:
+        return session.query(DagBundleModel).get(name)
```

Comment:
```suggestion
    @classmethod
    @provide_session
    def get(cls, name: str, session: Session = NEW_SESSION) -> DagBundleModel:
        return session.get(cls, name)
```

Is this function useful? It looks like the only advantage is we don’t need to create a session explicitly. But I only see this function used once, in which case just calling `create_session` when needed is actually less code…

Reply:
Good call. I'll refactor this tomorrow morning.

Reply:
In fact, we'd likely need to do this sooner than later anyways 👍